File: OxbridgeWebsite/src/app/models/event.ts
export class Event {
    constructor(
        public _id?:string, 
        public name?:string, 
        public eventStart?:string, 
        public eventEnd?:string, 
        public city?:string, 
        public eventCode?:string, 
        public eventStartTime?: string, 
        public eventEndTime?: string, 
        public isLive?:boolean, 
        public actualEventStart?: string
        ){}
}

File: prototypeproject/lib/Model/event.dart
class Event {
 
 String eventId, eventStart, eventEnd, city, eventCode, actualEventStart, isLive, name;

  
    Event({
        this.name,
        this.eventId,
        this.eventStart,
        this.eventEnd,
        this.city,
        this.eventCode,
        this.actualEventStart,
        this.isLive
      });

    factory Event.fromJson(Map<String, dynamic> json) {
    return Event(
      name:json['name'],
      eventId: json['_id'],
      eventStart: json['eventStart'],
      eventEnd: json['eventEnd'],
      city: json['city'],
      eventCode: json['eventCode'],
      actualEventStart: json['actualEventStart'],
      isLive: json['isLive'],
    );
  }
 
}

mixin required {
}